projects/plugins/jetpack/modules/masterbar/inline-help/inline-help.js

```diff
@@ -0,0 +1,16 @@
+( function () {
+	function init() {
+		var fauxInlineHelpButton = document.querySelector( '[data-faux-inline-help]' );
+		fauxInlineHelpButton &&
```

Comment:
Nitpick to satisfy linter but not a blocker.

```suggestion
		if ( fauxInlineHelpButton ) {
			fauxInlineHelpButton.addEventListener( 'click', function ( e ) {
				e.preventDefault();
				window.open( e.target.href, e.target.target );
			} );
		}
```